cmd/integrationArtifactGetMplStatus.go

```diff
@@ -106,6 +106,9 @@ func runIntegrationArtifactGetMplStatus(
 		if parsingErr != nil {
 			return errors.Wrapf(parsingErr, "HTTP response body could not be parsed as JSON: %v", string(bodyText))
 		}
+		if jsonResponse == nil {
+			return errors.Errorf("Empty json response: %v", string(bodyText))
```

Comment:
Shouldn't we use errors.Wrapf() to improve the error handling/logging?

https://github.com/SAP/jenkins-library/blob/0696f64ecd385403c8045d8c31bac89962b1037d/DEVELOPMENT.md#error-handling

Reply:
There is nothing to Wrap. I create a new error.